exporter/prometheusexporter/collector.go

```diff
@@ -207,6 +209,41 @@ func (c *collector) convertIntHistogram(metric pdata.Metric) (prometheus.Metric,
 	return m, nil
 }
 
+func (c *collector) convertSummary(metric pdata.Metric) (prometheus.Metric, error) {
+	dataPoints := metric.Summary().DataPoints()
+
+	var cumSum float64
+	var cumCount uint64
+	quantiles := make(map[float64]float64)
```

Comment:
Should these values be tracked on a per data point basis?  What is the expected behavior if a `pdata.Metric` is received that has multiple summary data points?  Perhaps I misunderstand the data model, but my assumption would be that they would all have quantile values at the same quantiles.  Should the most recent summary data point be exported?  It seems like the current implementation will aggregate the sum and count but take the last value seen for each quantile, which doesn't feel right.

Reply:
You do have a great point and thank you for articulating it! Indeed, one point with varying quantiles, single sum and single count. Thank you! Done.